blitz_api/serializers.py

```diff
@@ -501,7 +514,19 @@ def create(self, validated_data):
 
     class Meta:
         model = User
-        fields = '__all__'
+        fields = [
+            'id', 'url', 'email', 'first_name', 'last_name', 'is_active',
+            'phone', 'other_phone', 'is_superuser', 'is_staff', 'university',
+            'last_login', 'date_joined', 'academic_level', 'academic_field',
+            'gender', 'language', 'birthdate', 'groups', 'user_permissions',
+            'tickets', 'membership', 'membership_end', 'city',
+            'personnal_restrictions', 'academic_program_code', 'faculty',
+            'student_number', 'volunteer_for_workplace', 'hide_newsletter',
+            'is_in_newsletter', 'number_of_free_virtual_retreat',
+            'membership_end_notification', 'get_number_of_past_tomatoes',
+            'get_number_of_future_tomatoes', 'password',
+            'last_acceptation_terms_and_conditions',
+            'tomato_field_matrix']
```

Comment:
Would be better to be one by line only, it allow us to keep git change more cleaner and usefull for retro-analyse